generic/tests/cfg/iometer_windows.cfg

```diff
@@ -73,3 +73,52 @@
                     bg_mode = yes
                 - after_test:
                     bg_mode = no
+        - with_multi_scsi_disks:
+            only virtio_scsi
+            run_timeout = 3600
+            format_multi_disks = yes
+            images = "image1"
+            image_name_stg0 = images/storage0
+            image_name_stg1 = images/storage1
+            image_name_stg2 = images/storage2
+            image_name_stg3 = images/storage3
+            image_size_stg0 = 1G
+            image_size_stg1 = 1G
+            image_size_stg2 = 2G
+            image_size_stg3 = 1G
+            Win7, Win8, Win10, Win2008:
+                icf_name = "iometer_multi_2disks.icf"
+                disk_indexes = "1 2"
+                disk_letters = "I J"
+                disk_fstypes = "ntfs fat32"
+                images += " stg0 stg1"
+                force_create_image_stg0 = yes
+                force_create_image_stg1 = yes
+                remove_image_stg0 = yes
+                remove_image_stg1 = yes
+            Win2016, Win2019:
+                icf_name = "iometer_multi_3disks.icf"
+                disk_indexes = "1 2 3"
+                disk_letters = "I J K"
+                disk_fstypes = "ntfs fat32 refs"
+                images += " stg0 stg1 stg2"
+                force_create_image_stg0 = yes
+                force_create_image_stg1 = yes
+                force_create_image_stg2 = yes
+                remove_image_stg0 = yes
+                remove_image_stg1 = yes
+                remove_image_stg2 = yes
+            Win2012, Win2012..r2:
```

Comment:
Hi yhong, please update based that I updated support list secondly yesterday! 
    On win2012-64, win2012-r2, win2016-64, win2019-64, supported file system type as: FAT, FAT32, NTFS, ReFS. need to test 4 disk. each disk match a file system type.
    On Win7-32/64, Win8-32/64, Win8.1-32/64, Win10-32/64, Win2008-32/64/r2, supported file system type as: FAT, FAT32, NTFS. need to test 3 disk, each disk match with a file system type.
thus only need 2 iometer_**.icf file.
And suggest set all disk size to 2G, it match all 4 file system.

Reply:
ok, got it, will update it, thanks.